### ISSUE TITLE
`sizeof` within modules 

### ISSUE THREAD

**Issue by Naheel-Azawy** [vlang/v#5150]
**V version:** V 0.1.27 c4241f9
**OS:** 5.6.13-arch1-1

**What did you do?**
In my pull request #5149, inside the os module,
```
mut buf := [1024]byte
sizeof(buf)
```

**What did you expect to see?**
`sizeof(buf)` being normally 1024
 
**What did you see instead?**
`'os__buf' undeclared`

**Comment by JalonSolov** [comment]
Its "sort of" fixed in latest v.  The error message you were seeing no longer happens.  However...
```
jalon@Ripper:~/git/v$ v run foo.v
foo.v:1:5: warning: unused variable: `buf`
    1 | mut buf := [1024]byte
      |     ~~~
    2 | sizeof(buf)
    3 | println(sizeof(buf))
1024
jalon@Ripper:~/git/v$
```

**Comment by Naheel-Azawy** [comment]
Nice, that's some good progress!
Kinda off topic, I know the way it's implemented in C is different, but wouldn't it be nicer if `buf.len` was equivalent?

**Comment by JalonSolov** [comment]
Seems like it might, but .len currently only seems to work with structs, not arrays.

**Comment by spytheman** [comment]
.len is available for ordinary arrays, not fixed ones.

**Comment by spytheman** [comment]
Fixed in #5855
